src/apps/pillarx-app/components/WalletPortfolioButtons/WalletPortfolioButtons.tsx

```
@@ -38,23 +44,157 @@ const WalletPortfolioButtons = () => {
     !isPrivyConnectedViaWalletConnect &&
     kit.getEtherspotProvider().getWalletMode() !== 'delegatedEoa';
 
+  const handleAddCash = async () => {
+    if (isAddCashLoading) return;
+
+    setIsAddCashLoading(true);
+
+    try {
+      // Validate wallet address
+      if (!accountAddress) {
+        console.error('Wallet address is not available');
```

Does this result in a silent error?

---

yes will add alert on these